static/templates/task/base.html

```diff
@@ -1,4 +1,4 @@
-<div class="_main-content" style="width: 100%">
```

Comment:
can I revert the file in the browser interface for github or do I have to make another PR after editing in my text editor?